### PR TITLE
Transfer Components + Api Integration

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#13609

*Description of changes:*
* Updates HomeOwners components to be compatible with Transfer Flow
* Implements basic API integration with new Transfer Endpoint
* Feedback updates with regards to Breadcrumb and header listed in ticket

*Notes:* - Some data (such as Submitting Party address) is not collected by the UI and subsequently mocked in the request to satisfy the Schema

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by ChasBoucher — APPROVED** [review]
Looks Good.

File: ppr-ui/src/utils/mhr-api-helper.ts [hunk 1]
```diff
@@ -282,3 +282,24 @@ export async function validateDocumentID (documentId: string) {
     }
   }
 }
+
+export async function submitMhrTransfer (payloadData, mhrNumber) {
+  const url = sessionStorage.getItem('MHR_API_URL')
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
```
Comment (dimak1) [inline ppr-ui/src/utils/mhr-api-helper.ts:288]: For the POST `/mhr/api/v1/transfers/{mhrNumber}` api call, **Account-Id** is required in the headers.

**Review by dimak1 — APPROVED** [review]
Nice and clean implementation! Just one comment for the POST call.

File: ppr-ui/src/utils/mhr-api-helper.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/mhr-api-helper.ts:288]: Oh! I didn't see, do you have a reference?

Comment (dimak1) [reply ppr-ui/src/utils/mhr-api-helper.ts:288]: Sent you via RocketChat.

Comment (doug-lovett) [reply ppr-ui/src/utils/mhr-api-helper.ts:288]: Following the ppr api I think there is an axios request setup that tries to add the account id in the header for all requests?

**Review by doug-lovett — APPROVED** [review]
Looks good, just a note that using a group id of 1 only works if the MH registration has no previous transfers. The group id increments as changes are made. I suggest testing  in this sprint with a new sole owner registration.

File: ppr-ui/src/utils/mhr-api-helper.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/mhr-api-helper.ts:288]: @dimak1 Sorry, to follow up on this, the `account-id` IS included. 
It's handled by the axios interceptor `axios-ppr.ts`. It is conditionally included in header config when there is a `currentAccount` present. ie any non-staff, for the most part. 

Good eye though, thanks for noting this!

**Comment by cameron-eyds** [comment]
> Looks good, just a note that using a group id of 1 only works if the MH registration has no previous transfers. The group id increments as changes are made. I suggest testing in this sprint with a new sole owner registration.



> Looks good, just a note that using a group id of 1 only works if the MH registration has no previous transfers. The group id increments as changes are made. I suggest testing in this sprint with a new sole owner registration.

@doug-lovett Good call, thanks will do!
 I hadn't done extensive testing with the groups aspect of Home Owners yet, so i'm sure we can flesh this out further in the next iteration.

File: ppr-ui/src/utils/mhr-api-helper.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/utils/mhr-api-helper.ts:288]: @doug-lovett Oh, you beat me to it 😆 
Thank you!

**Pull request merged**